docs/FAQ.md

```diff
@@ -120,6 +122,14 @@ It is also important to remember, that even if your contact enabled "Delete for
 
 When "Delete for everyone" is not enabled, you can still mark the sent message as deleted within 24 hours of sending it. In this case the recipient will see it as "deleted message", and will be able to reveal the original message.
 
+### Why invite links use simplex.chat domain?
+
+You can replace `https://simplex.chat/` with `simplex:/` or with any other domain - the app never connect with it, ignoring it completely. It is only used to make it easier to connect for the new users who did not install the app yet.
+
+The invitation links will soon move to servers' domains. The servers already can host the pages that will be used to show QR codes.
+
+The link itself and the key exchange are not hosted anywhere, and the server that hosts the page to show QR code does not observe the actual connection link, because it is in the hash part of the link. The part after hash character (`#`) is not sent over the internet - the server can only see https://simplex.chat/contact/ and the rest is processed on user's device in the browser, if you open it as a page.
```

Comment:
```suggestion
The link itself and the key exchange are not hosted anywhere, and the server that hosts the page to show QR code does not observe the actual connection link, because it is in the hash part of the link. The part after hash character (`#`) is not sent over the internet - the server can only see `https://simplex.chat/contact/` and the rest is processed on user's device in the browser, if you open it as a page.
```